### ISSUE TITLE
How to update keepassxc?

### ISSUE THREAD

**Issue by Spiralpine** [keepassxreboot/keepassxc#7517]
Hello - 

Linux novice here. I've installed keepass xc on my chromebook using the terminal commands for Linux (sudo apt-get install keepassxc). The GUI shows I'm currently at  2.6.2. 

Tried to update by:
sudo apt-get update && sudo apt-get upgrade keepassxc

but, it does not update to the latest 2.6.6 version. It gives me this:

Hit:1 https://security.debian.org bullseye-security InRelease
Hit:2 https://brave-browser-apt-release.s3.brave.com stable InRelease           
Hit:3 https://updates.signal.org/desktop/apt xenial InRelease                   
Hit:4 https://deb.debian.org/debian bullseye InRelease                          
Ign:5 https://storage.googleapis.com/cros-packages/99 bullseye InRelease       
Hit:6 https://storage.googleapis.com/cros-packages/99 bullseye Release
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
keepassxc is already the newest version (2.6.2+dfsg.1-1).
Calculating upgrade... Done
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

How do I update to the latest version?

Thanks in advance.

**Comment by droidmonkey** [comment]
https://keepassxc.org/download

**Comment by Spiralpine** [comment]
> https://keepassxc.org/download

The installation is done via the Debian repository that Chromebook uses. And, looks like that repository has no update beyond 2.6.2.

**Comment by Spiralpine** [comment]
Is there a suitable KeepassXC package on github for the Linux container that runs on Chromebooks that I can download? If so, any instructions on how to install it?

Thanks in advance.

**Comment by droidmonkey** [comment]
No we do not support chromebooks

**Comment by Spiralpine** [comment]
> No we do not support chromebooks

It runs perfectly on the "linux container" in my Chromebook using the Debian package. It's really the Debian support which is also on https://keepassxc.org/download/#linux

Let me rephrase my previous question:

Is there a suitable KeepassXC package on github for the Debian Linux that I can download? If so, any instructions on how to install it?

**Comment by droidmonkey** [comment]
Use the appimage, download, make it executable,  double click to run.

**Comment by Spiralpine** [comment]
You mean the KeePassXC-2.6.6-x86_64.AppImage? Is that going to work for Linux (64bit Intel)?

**Comment by droidmonkey** [comment]
Give it a try, you are on Linux now, the guard rails are removed 😉

**Comment by Spiralpine** [comment]
> Give it a try, you are on Linux now, the guard rails are removed 😉

Thanks so much. It worked wonderfully. Don't take the appimage download option away  :-) ... so handy

**Comment by waaweech** [comment]
Also, tried to verify the signature for the AppImage and I'm getting this:

user1@penguin:~/Applications/keepassxc-appimage$ gpg --verify --verbose --keyring vendors.gpg ./KeePassXC-2.6.6-x86_64.AppImage.sig
gpg: assuming signed data in './KeePassXC-2.6.6-x86_64.AppImage'
gpg: Signature made Sat 12 Jun 2021 09:41:51 AM EDT
gpg:                using RSA key C1E4CBA3AD78D3AFD894F9E0B7A66F03B59076A8
gpg: Note: signature key AFF235EEFB5A2517 expired Thu 07 Jan 2021 01:49:30 PM EST
gpg: Note: signature key AFF235EEFB5A2517 has been revoked
gpg: using subkey B7A66F03B59076A8 instead of primary key CFB4C2166397D0D2
gpg: Note: signature key AFF235EEFB5A2517 expired Thu 07 Jan 2021 01:49:30 PM EST
gpg: Note: signature key AFF235EEFB5A2517 has been revoked
gpg: using subkey B7A66F03B59076A8 instead of primary key CFB4C2166397D0D2
gpg: using pgp trust model
gpg: Good signature from "KeePassXC Release <release@keepassxc.org>" [unknown]
gpg: Note: signature key AFF235EEFB5A2517 expired Thu 07 Jan 2021 01:49:30 PM EST
gpg: Note: signature key AFF235EEFB5A2517 has been revoked
gpg: using subkey B7A66F03B59076A8 instead of primary key CFB4C2166397D0D2
gpg: Note: signature key AFF235EEFB5A2517 expired Thu 07 Jan 2021 01:49:30 PM EST
gpg: Note: signature key AFF235EEFB5A2517 has been revoked
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: BF5A 669F 2272 CF43 24C1  FDA8 CFB4 C216 6397 D0D2
     Subkey fingerprint: C1E4 CBA3 AD78 D3AF D894  F9E0 B7A6 6F03 B590 76A8
gpg: binary signature, digest algorithm SHA512, key algorithm rsa2048
